src/Abp.RedisCache/Runtime/Caching/Redis/DefaultRedisCacheSerializer.cs

```diff
@@ -14,31 +14,23 @@ public class DefaultRedisCacheSerializer : IRedisCacheSerializer, ITransientDepe
         /// <summary>
         ///     Creates an instance of the object from its serialized string representation.
         /// </summary>
-        /// <param name="objbyte">String representation of the object from the Redis server.</param>
+        /// <param name="redisValue">String representation of the object from the Redis server.</param>
         /// <returns>Returns a newly constructed object.</returns>
         /// <seealso cref="IRedisCacheSerializer.Serialize" />
-        public virtual object Deserialize(RedisValue objbyte)
+        public virtual T Deserialize<T>(RedisValue redisValue)
```

Comment:
I think we can keep `object` and leave the type casting to higher level, for example AbpRedisCache/ProtoBufRedisCache

Reply:
hi @ryancyq 
This pr will be refactored when the cache supports generics. : )

Reply:
👍